apps/client/src/features/app-settings/panel/project-panel/ProjectPanel.module.scss

```diff
@@ -50,3 +54,9 @@
   flex-direction: column;
   gap: 1em;
 }
+
+.toggleOption {
+  display: flex;
+  flex-direction: row;
+  gap: 1em;
+}
```

Comment:
we can remove this and add the class `toggleOption` to the CSS above that contains equivalent styling